### PR TITLE
feat(widget-message-meet): Update widget data api

### PR DESCRIPTION
**Pull request by bzang**
To use the new code, you'll need to change the `USERID` environment var in `.env` to `TO_USER_EMAIL`.

### CODE REVIEW

File: packages/widget-message-meet/README-USAGE.md [hunk 1]
```diff
@@ -81,28 +81,32 @@ The easiest way to get the Spark Chat Widget into your web site is to add the bu
   -  Add a `<link />` tag to include `main.css`
 1.  Create a container where you would like to embed the chat widget and add the following attributes to configure the widget:
   - `data-toggle="spark-chat"`: (required)
```
Comment (adamweeks) [inline packages/widget-message-meet/README-USAGE.md]: Should we change `chat` here?

File: packages/widget-message-meet/src/components/list-separator/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ function ListSeparator(props) {
 }
 
 ListSeparator.propTypes = {
-  isInformative: PropTypes.boolean,
+  isInformative: PropTypes.bool,
```
Comment (adamweeks) [inline packages/widget-message-meet/src/components/list-separator/index.js]: Yea, didn't catch that one before, got it fixed in my branch too.

File: packages/widget-message-meet/README-USAGE.md [hunk 3]
```diff
@@ -81,28 +81,32 @@ The easiest way to get the Spark Chat Widget into your web site is to add the bu
...
-  - `data-access-token`: Access token for the user account initiating the messaging session. For testing purposes you can use a developer access token from <https://developers.ciscospark.com>
-  - `data-user-id`: User Id or email of the message recipient.
+  - `data-access-token`: (required) Access token for the user account initiating the messaging session. For testing purposes you can use a developer access token from <https://developers.ciscospark.com>.
+  - Include one of the following attributes:
+    - `data-to-person-email`: Email of the message recipient.
+    - `data-to-person-id`: User Id of the message recipient.
 
     ```html
     <div class="chat-widget-container"
```
Comment (bzang) [inline packages/widget-message-meet/README-USAGE.md]: Good catch

File: packages/widget-message-meet/Gruntfile.js [hunk 4]
```diff
@@ -52,6 +52,6 @@ module.exports = function configGrunt(grunt) {
 
   grunt.registerTask('test', ['jest']);
   grunt.registerTask('test-clean', ['clean:snapshots', 'jest']);
-  grunt.registerTask('serve', ['webpack-dev-server:start']);
-  grunt.registerTask('default', ['serve']);
+  grunt.registerTask('start', ['webpack-dev-server:start']);
```
Comment (ianwremmel) [inline packages/widget-message-meet/Gruntfile.js]: Can you alias serve onto start? Ideally, all packages will expose "build", "test", and "serve".

**Comment by bzang** [comment]
Serve gets overwritten by a lot level alias. I've tried and our scripts never get run. Haven't figured out how to do the override correctly yet. That's why its using a different alias.

**Comment by ianwremmel** [comment]
gotcha. line 411 of Gruntfile.package.json needs to be about 10 lines lower.

**Pull request merged**